std_msgs/Header header
geometry_msgs/Pose pose
deformable_astar/DeformationState properties
